cli/concept.md

```diff
@@ -0,0 +1,157 @@
+# Overview
+The Amplify cli enables front-end developers to easily set up the backend resources in the cloud. <br/>
+It is designed to work side by side with the [Amplify](https://github.com/aws-amplify/amplify-js) library. Official resource categories provided by the cli can be easily consumed by the corresponding Amplify library modules. <br/>
```

Comment:
replace with "Resources in your AWS account that the Amplify CLI category commands create can be easily consumed by the corresponding Amplify library modules or native SDKs."